src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,73 @@
+UART Device Emulation Module (DEM-UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module connects to the bus of a given CPU on one side, and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550A device, but instead of transmitting information over a UART-Interface it instead passes it to a host PC via the Debug Interconnect.
+
+.. _uart_registers:
+
+16550 UART Registers
+--------------------
+The following UART registers are implemented and accessible via the bus, the address mapping is in accordance with the UART-16550(A) standard as specified in this `Datasheet <http://caro.su/msx/ocm_de1/16550.pdf>`_.
+Currently there are no FIFOs present in the hardware, therefore writing to the ``FCR`` has no effect.
+The only bit in the ``LCR`` that has any meaning is the DLAB aka ``LCR7``, all other bits hold their written value but have no meaning.
```

Comment:
If you include the specifics down in the table with the register description, you can reduce these two sentences to "No FIFOs are present in hardware."